docs/gherkin.md

```diff
@@ -0,0 +1,112 @@
+# Gherkin
+
+Gherkin is the language that Cucumber understands. It is a Business Readable, [Domain Specific Language](http://martinfowler.com/bliki/BusinessReadableDSL.html) that lets you describe software’s behaviour without detailing how that behaviour is implemented.
+
+Gherkin serves two purposes — documentation and automated tests. The third is a bonus feature — when it yells in red it’s talking to you, telling you what code you should write.
+
+Gherkin’s grammar is defined in the Treetop grammar that is part of the Cucumber codebase. The grammar exists in different flavours for many spoken languages (37 at the time of writing), so that your team can use the keywords in your own language.
+
+There are a few conventions.
+
+ * Single Gherkin source file contains a description of a single feature.
+ * Source files have `.feature` extension.
+
+## Gherkin Syntax
+
+Like Python and YAML, Gherkin is a line-oriented language that uses indentation to define structure. Line endings terminate statements (eg, steps). Either spaces or tabs may be used for indentation (but spaces are more portable). Most lines start with a keyword.
```

Comment:
Those three first paragraphs are a bit tough to digest, especially for beginners. I suggest we move those specificities down (or even drop them for now) and simply explain that .feature files are text files with a limited number of keywords and a lot of content specific to the user's domain: descriptions, business rules and examples?